ChaosMod/Effects/db/Player/PlayerAutopilot.cpp

```diff
@@ -68,7 +68,7 @@ static void OnTick()
 
 #ifdef _DEBUG
 	// Draw debug go to line
-	if (m_state == STATE_TO_COORDS)
+	if (ms_State == STATE_TO_COORDS)
```

Comment:
Whilst correct, this is out of the scope of the PR and should be done in a separate commit.

Reply:
This change is needed for compilation in debug mode. I will take it out once all issues are resolved and this pull request is ready for merging.